alacritty_terminal/src/ansi.rs

```diff
@@ -363,65 +363,65 @@ impl Default for CursorStyle {
     }
 }
 
-/// Terminal modes
+/// Terminal modes.
 #[derive(Debug, Eq, PartialEq)]
 pub enum Mode {
-    /// ?1
+    /// ?1.
     CursorKeys = 1,
-    /// Select 80 or 132 columns per page
+    /// Select 80 or 132 columns per page.
     ///
-    /// CSI ? 3 h -> set 132 column font
-    /// CSI ? 3 l -> reset 80 column font
+    /// CSI ? 3 h -> set 132 column font.
+    /// CSI ? 3 l -> reset 80 column font.
     ///
     /// Additionally,
     ///
     /// * set margins to default positions
     /// * erases all data in page memory
     /// * resets DECLRMM to unavailable
-    /// * clears data from the status line (if set to host-writable)
+    /// * clears data from the status line (if set to host-writable).
     DECCOLM = 3,
-    /// IRM Insert Mode
+    /// IRM Insert Mode.
     ///
-    /// NB should be part of non-private mode enum
+    /// NB should be part of non-private mode enum.
     ///
-    /// * `CSI 4 h` change to insert mode
-    /// * `CSI 4 l` reset to replacement mode
+    /// * `CSI 4 h` change to insert mode.
+    /// * `CSI 4 l` reset to replacement mode.
     Insert = 4,
-    /// ?6
+    /// ?6.
     Origin = 6,
-    /// ?7
+    /// ?7.
     LineWrap = 7,
-    /// ?12
+    /// ?12.
     BlinkingCursor = 12,
-    /// 20
+    /// 20.
     ///
     /// NB This is actually a private mode. We should consider adding a second
     /// enumeration for public/private modesets.
     LineFeedNewLine = 20,
-    /// ?25
+    /// ?25.
     ShowCursor = 25,
-    /// ?1000
+    /// ?1000.
     ReportMouseClicks = 1000,
-    /// ?1002
+    /// ?1002.
     ReportCellMouseMotion = 1002,
-    /// ?1003
+    /// ?1003.
     ReportAllMouseMotion = 1003,
-    /// ?1004
+    /// ?1004.
     ReportFocusInOut = 1004,
-    /// ?1005
+    /// ?1005.
     Utf8Mouse = 1005,
-    /// ?1006
+    /// ?1006.
     SgrMouse = 1006,
-    /// ?1007
+    /// ?1007.
     AlternateScroll = 1007,
-    /// ?1049
+    /// ?1049.
     SwapScreenAndSetRestoreCursor = 1049,
-    /// ?2004
+    /// ?2004.
     BracketedPaste = 2004,
```

Comment:
In formatted comments like this it seems counter-productive to add a period at the end.

Wouldn't want people to think it's `?1049.` as escape. If anything, we should do something more like:

At the top there's also a list of things, we shouldn't add periods there.